pootle/apps/pootle_statistics/models.py

```diff
@@ -551,6 +551,13 @@ def get_similarity(self):
     def is_similarity_taken_from_mt(self):
         return self.submission.similarity < self.submission.mt_similarity
 
+    def get_suggested_words(self):
```

Comment:
@ta2-1 `_words` vs. `_wordcount` carry very different meanings (**actual** words vs. a **number** of words). Whatever the outcome is, this is not about preferences, but about accuracy and explicitness.